### DISCUSSION TITLE
是否考虑为openwrt提供裸的可执行文件

### DISCUSSION THREAD

**Discussion by zhang616123** [messense/aliyundrive-webdav#192] [Q&A] [answered]
ipk安装包对很多路由器来说过大，能否提供裸可执行文件，这样可以通过脚本在/tmp目录中运行。

**Comment by messense** [comment, marked as answer]
本来就有：https://github.com/messense/aliyundrive-webdav/releases/tag/v1.0.0

**Reply by zhang616123** [reply]
> 本来就有：https://github.com/messense/aliyundrive-webdav/releases/tag/v1.0.0

不好意思，似乎疏忽了，抱歉打开了这个问题
